WDE.SmartScriptEditor/Editor/Views/SmartDataEditors/SmartDataTargetsEditorView.xaml

```diff
@@ -0,0 +1,123 @@
+            <TextBox Grid.Column="1" Grid.Row="0" Text="{Binding Source.NameReadable, Mode=TwoWay}" TextAlignment="Left" VerticalAlignment="Center" />
+        </Grid>
+        <Separator Margin="0, 10, 10, 10" />
+        <Grid>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="120" />
+                <ColumnDefinition />
+            </Grid.ColumnDefinitions>
+            <Grid.RowDefinitions>
+                <RowDefinition Height="70" />
+            </Grid.RowDefinitions>
+            <Label Grid.Column="0" Grid.Row="0" Content="Target Description:" VerticalAlignment="Center" />
+            <ScrollViewer Grid.Column="1" Grid.Row="0">
+                <TextBox Text="{Binding Source.Description, Mode=TwoWay}" TextWrapping="Wrap" 
+                     VerticalAlignment="Stretch" HorizontalAlignment="Stretch"/>
+            </ScrollViewer>
+        </Grid>
+        <Separator Margin="0, 10, 10, 10"/>
+        <Grid>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="80"/>
+                <ColumnDefinition />
+            </Grid.ColumnDefinitions>
+            <Label Grid.Column="0" Grid.Row="0" Content="Help Text:" />
+            <TextBox Grid.Column="1" Grid.Row="0" Text="{Binding Source.Help, Mode=TwoWay}"/>
+        </Grid>
+        <Separator Margin="0, 10, 10, 10" />
+        <ListView ItemsSource="{Binding TargetTypes}" MaxHeight="100">
+            <ListView.ItemTemplate>
+                <DataTemplate>
+                    <CheckBox Content="{Binding Name}" IsChecked="{Binding IsChecked, Mode=TwoWay}"
+                              HorizontalAlignment="Stretch" />
+                </DataTemplate>
+            </ListView.ItemTemplate>
+            <ListView.ItemContainerStyle>
+                <Style TargetType="{x:Type ListViewItem}">
+                    <Setter Property="Focusable" Value="False" />
+                </Style>
+            </ListView.ItemContainerStyle>
+        </ListView>
+        <Separator Margin="0, 10, 10, 10" />
+        <Label Content="Parameters:" />
+        <Button VerticalContentAlignment="Center" HorizontalContentAlignment="Center" Command="{Binding AddParameter}" 
+                Width="25" Height="25" VerticalAlignment="Center" HorizontalAlignment="Left" Margin="10, 0, 0, 10">
+            <Path HorizontalAlignment="Center" VerticalAlignment="Center" Stroke="{DynamicResource {x:Static SystemColors.ControlTextBrushKey}}"
+                  Data="M3.875,0 L5.125,0 5.125,3.875 9,3.875 9,5.125 5.125,5.125 5.125,9 3.875,9 3.875,5.125 0,5.125 0,3.875 3.875,3.875 3.875,0 z" />
+        </Button>
+        <ListView MinHeight="100" ItemsSource="{Binding Source.Parameters}" SelectedIndex="{Binding SelectedParamIndex}">
+            <ListView.ItemTemplate>
+                <DataTemplate>
+                    <Grid>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition />
```

Comment:
Could you set MinWidth here? Something around 100 units maybe? To make it looks more like a table